.idea/gradle.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
seems like you've pushed files from `.idea` folder... try to fix that with `rebase + edit` operation